pkg/agent/knapsack/knapsack.go

```diff
@@ -362,11 +362,11 @@ func (k *knapsack) TraceSamplingRate() float64 {
 	return k.flags.TraceSamplingRate()
 }
 
-func (k *knapsack) SetObservabilityIngestServerURL(url string) error {
-	return k.flags.SetObservabilityIngestServerURL(url)
+func (k *knapsack) SetTraceIngestServerURL(url string) error {
+	return k.flags.SetTraceIngestServerURL(url)
 }
-func (k *knapsack) ObservabilityIngestServerURL() string {
-	return k.flags.ObservabilityIngestServerURL()
+func (k *knapsack) TraceIngestServerURL() string {
+	return k.flags.TraceIngestServerURL()
 }
 
 func (k *knapsack) SetDisableObservabilityIngestTLS(enabled bool) error {
```

Comment:
These should probably all get renamed to `SetDisableTraceIngestTLS` too?

Reply:
done